bugbot/rules/workflow/no_severity.py

```diff
@@ -277,6 +279,23 @@ def get_bz_params(self, date):
                 }
             )
 
+        # TODO: the following code can be removed in 2023.
+        # https://github.com/mozilla/bugbot/issues/1596
```

Comment:
I guess we should reopen the issue?

Reply:
Done! Based on the analyses in https://github.com/mozilla/bugbot/issues/1596#issuecomment-1583610156. It may be better to expand a month every day instead of a month every week. WDYT?

Reply:
Sounds good to me, even more than a month every day would work

Reply:
Done in 550dd042142de019d8c6a372e3fb3c1d2c17e844.